### ISSUE TITLE
Unable to run Contrack 

### ISSUE THREAD

**Issue by njhunsak** [vistalab/vistasoft#153]
I'm trying to use contrack and continue to run into errors:

```bash
-bash-4.1$ /fslhome/intj5/apps/matlab/vistasoft/mrDiffusion/fiber/tractography/contrack/contrack_gen.glxa64 -i ~/compute/data/MIOS/1004--2014-08-15/fibers/conTrack/ctrSampler_20160106T184548.txt -p ~/compute/data/MIOS/1004--2014-08-15/fibers/conTrack/fx1_fx2_20160106T184548.pdb
Seed(s): 1452178977
Fetching parameters from disk ... (file: /fslhome/intj5/compute/data/MIOS/1004--2014-08-15/fibers/conTrack/ctrSampler_20160106T184548.txt)
Loading Track Params File
Fetching PDF volume from disk...(file: /fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/pdf.nii.gz)
Volume dimensions: 81 x 106 x 76
Fetching WM/GM from disk...(file: /fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/wmProb.nii.gz)
Fetching VOI mask volume from disk...(file: /fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/fx1_fx2_20160106T184548.nii.gz)
** ERROR (nifti_image_read): failed to find header file for '/fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/fx1_fx2_20160106T184548.nii.gz'
Error reading VOI volume, continuing without.
Sampling pathways ... 0% done.
Tracker failing because there is no place to start!
```

This is the system I am trying to run this on:

```bash
Linux version 2.6.32-504.16.2.el6.x86_64 (mockbuild@x86-028.build.eng.bos.redhat.com) (gcc version 4.4.7 20120313 (Red Hat 4.4.7-9) (GCC) ) #1 SMP Tue Mar 10 17:01:00 EDT 2015
```

**Comment by lmperry** [comment]
It appears that there is an issue with your ROI file. The nifti reader is failing to read its header. Can you confirm that it is a valid file?

Also, the pre-compiled contrack binary was compiled for Ubuntu Linux.

**Comment by njhunsak** [comment]
Thanks for your quick reply, I've done some digging, but I've definitely hit a road block:

I was able to confirm the error has nothing to do with the images themselves. I was able to run an old version of contrack (circa 2009) on this system: Linux version 2.6.18-194.26.1.el5 (gcc version 4.1.2 20080704 (Red Hat 4.1.2-48)). I was able to run both the fiber generation and the fiber scoring software; however, I have not been able to get the software to run on any other computer:

On the Linux v2.6.32 machine:
1. When I run the latest version of contrack_gen from the github repository I get the error: "Tracker failing because there is no place to start!"
2. When I run the contrack version I moved over from the previous Linux box, I get the same error: "Tracker failing because there is no place to start!"
3. When I run the version located under "mrDiffusion/fiber/tractography/contrack/old/contrack_gen.glxa64", the program gets stuck at "Loading Tract Params File".

If I try to run on a Mac machine (I've tested this on a MacBook Pro 2.5 GHz Intel Core i7, Mac Pro Early 2009 Quad-Core Intel Xeon, and Mac Pro Late 2013 all running the latest OS), I'm able to sort of get contract_gen to run, but get the follow error with the default settings:

```bash
fb5PDF: 1.70539e-07watsonPDF: 30.7105
k1: -132.701 k2: -6.43451 log_A: -4.01701
e1: 0.953683 , -0.197193 , 0.227164
e2: 0.245668 , 0.946364 , -0.209861
Still rejecting samples after 10000 attempts
```
If I change the `Save Out Spacing: 1` in the ctrSampler file, I'm able to mostly run the program without the above error. Regardless of whether or not I can get contrack_gen to run on any of the mac computers, when I try to run contrack_score, I get a whole new set of errors that say I've exceeded my memory load, which doesn't make sense given that all my newer systems are far better than my older system. Again, when I ran this on the old system with the old version, I was easily able to run a test brain. Taking that same participant, I am unable to run it on any of these new systems with any of the versions of contrack that I have.

**Comment by wandell** [comment]
Please send us a picture confirming that you can read the VOI nifti file. The error message is pretty clear.  And of course, if you can't read that specific file, the rest will fail.  Once you show that the VOI can be read, we can go on to the next question.

Fetching VOI mask volume from disk...(file: /fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/fx1_fx2_20160106T184548.nii.gz)
** ERROR (nifti_image_read): failed to find header file for '/fslhome/intj5/compute/data/MIOS/1004--2014-08-15/dt6dir/bin/fx1_fx2_20160106T184548.nii.gz'
Error reading VOI volume, continuing without.

**Comment by njhunsak** [comment]
I think I figured out how to fix the first error. I needed to play around with my ROIs a bit. I'm still getting the fb5PDF error, but the fiber track is generated and I was able to score it, so I'm not sure if I should worry about the error.

<img width="1440" alt="screen shot 2016-01-08 at 4 34 40 pm" src="https://cloud.githubusercontent.com/assets/4333838/12238514/97470afe-b841-11e5-801e-fd6027353d2f.png">
